### PR TITLE
Fixes for Local- and RemoteObjectRepo - with fixed CI

### PR DESCRIPTION
**Pull request by NikkelM**
For the same reasons as PR #105, I have branched main again and copied over the changes from #103. I have also added two tests to check the correct functionality of ifPresent/ifAbsent-blocks in the Repos.

### CODE REVIEW

File: packages/Liquid-Tests.package/ObjectRepoTests.class/instance/testifPresentIfAbsentRemote.st [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	client 
+		at: 'newPoll'
+		ifAbsent: [self fail].
+	client 
+		at: 'newPoll'
+		ifPresent: [presentCheck := true]
+		ifAbsentPut: ThrowerOfErrors new. 	
+		
+	self assert: presentCheck equals: true.
+
```
Comment (Skn0tt) [inline packages/Liquid-Tests.package/ObjectRepoTests.class/instance/testifPresentIfAbsentRemote.st:22]:
```suggestion
    server destroy
```
[pebble] This makes sure the server doesn't take up a spare port.

File: packages/Liquid-Tests.package/ObjectRepoTests.class/instance/testifPresentIfAbsentRemote.st [hunk 2]
```diff
@@ -0,0 +1,22 @@
+tests
+testifPresentIfAbsentRemote
+	| server port url client presentCheck|
+	presentCheck := false.
+	port := 10000 random.
+	server := RemoteRepoServer new listenOn: port.
+	url := 'http://localhost:' , port.
+	client := RemoteRepoClient new url: url.
```
Comment (Skn0tt) [inline packages/Liquid-Tests.package/ObjectRepoTests.class/instance/testifPresentIfAbsentRemote.st:8]: [sand] This test is very similar to the one with the LocalObjectRepo - maybe we can extract the common body into a method and inject the `LocalObjectRepo` / `RemoteRepoClient` instance via parameters?

**Review by Skn0tt — CHANGES_REQUESTED** [review]

File: packages/Liquid-Tests.package/ObjectRepoTests.class/instance/testifPresentIfAbsentRemote.st (hunk 1 above)
Comment (NikkelM) [reply packages/Liquid-Tests.package/ObjectRepoTests.class/instance/testifPresentIfAbsentRemote.st:22]: Thanks for the info. I have included this change with the other changes in the next commit

File: packages/Liquid-Tests.package/ObjectRepoTests.class/instance/testifPresentIfAbsentRemote.st (hunk 2 above)
Comment (NikkelM) [reply packages/Liquid-Tests.package/ObjectRepoTests.class/instance/testifPresentIfAbsentRemote.st:8]: Good thought. I have set up setUp and tearDown methods that manage this and the `server destroy`

Comment (Skn0tt) [reply packages/Liquid-Tests.package/ObjectRepoTests.class/instance/testifPresentIfAbsentRemote.st]: Nice! We still have some duplicated code in the remote / local test, can we extract them in some method?

Comment (NikkelM) [reply packages/Liquid-Tests.package/ObjectRepoTests.class/instance/testifPresentIfAbsentRemote.st]: I have combined the two tests into one in the next commit

**Review by Skn0tt — APPROVED** [review]

**Pull request merged**